internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,324 @@
+package user
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestDisableSetup(t *testing.T) {
+	t.Run("should construct a realm client with the configured base url", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		profile.SetRealmBaseURL("http://localhost:8080")
+		cmd := &CommandDisable{inputs: disableInputs{}}
+
+		assert.Nil(t, cmd.realmClient)
+		assert.Nil(t, cmd.Setup(profile, nil))
+		assert.NotNil(t, cmd.realmClient)
+	})
+}
+
+func TestDisableHandler(t *testing.T) {
```

Comment:
[nit] keep the test func naming pattern consistent here and use `TestUserDisable*`